airgun/views/hardware_model.py

```diff
@@ -18,7 +18,7 @@ class HardwareModelsView(BaseLoggedInView, SearchableViewMixinPF4):
         './/table',
         column_widgets={
             'Name': Text('.//a'),
-            'Actions': Button('Delete'),
+            4: Pf4SvgActionsDropdown(".//div[contains(@class, 'pf-c-dropdown')]"),
```

Comment:
Can you please try out this widget rather writing new widget https://github.com/SatelliteQE/airgun/blob/master/airgun/views/acs.py#L362

Reply:
Thank you for the hint. This was the abstraction I was looking for but couldn't not find.